src/pages/TrueSightV2/components/chart/index.tsx

```diff
@@ -2814,3 +2814,58 @@ export const Prochart = ({
     </ProLiveChartWrapper>
   )
 }
+
+/* IN DEVELOPMENT */

```

Comment:
what is this

Reply:
phần này có design UI rồi mà chưa có api hay chốt gì từ phía BE nên để tạm vậy